src/topp/IDFileConverter.cpp

```diff
@@ -327,12 +329,29 @@ class TOPPIDFileConverter :
           MascotXMLFile::initializeLookup(lookup, exp, scan_regex);
           PepXMLFile().load(in, protein_identifications,
                             peptide_identifications, exp_name, lookup);
+          bool reset_basename = getFlag_("reset_basename");
+          bool add_ions = getFlag_("add_ionmatch_annotation");
+          if (add_ions)
+          {
+            SpectrumMetaDataLookup::addMissingSpectrumReferencestoPeptideIDs(
+                      peptide_identifications, exp_name, false, add_ions, reset_basename);
+          }
```

Comment:
reannotation without knowing details on fragmentation method / tolerances does make very litte sense. Here you would have this information available and could call e.g. a method from a not-ecisting SpectrumReannotation .class.